flink-connectors/flink-connector-hive/pom.xml

```
@@ -59,6 +59,12 @@ under the License.
 			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
+                <dependency>
+                        <groupId>org.apache.flink</groupId>
+                        <artifactId>flink-table-api-java-bridge_2.11</artifactId>
```

```suggestion
                        <artifactId>flink-table-api-java-bridge_${scala.binary.version}</artifactId>
```

BTW, since we depend on flink-table-api-java-bridge now, we can probably just remove the above dependency of flink-table-api-java